apps/web/src/pages/Pools/EditPoolPage/EditPoolPage.tsx

```diff
@@ -682,6 +698,12 @@ export const EditPoolForm = ({
                       onSave={onSave}
                       onUpdatePublished={onUpdatePublished}
                     />
+                    <ContactEmailSection
```

Comment:
Is it just me?  The input on this doesn't let me type when the pool is published but you can still submit.  But it doesn't save properly?

Reply:
I found that issue as well. Fixed in [Fix contact email input in edit pool page](https://github.com/GCTC-NTGC/gc-digital-talent/pull/14445/commits/ba7204e3f1ce317e2c8a8760a55cecc9ee584656)